### PR TITLE
WIP events/React docs

### PR DESCRIPTION
**Pull request by LarryUllman**
Starts editing the event/React docs for clarity and consistency. Still need to do the "Use automations for dynamic responses" page. 

### Checklist
- [x] This pull request includes a label categorizing the change e.g. `maintenance`, `fix`, `feature`, `enhancement`, `docs`.

For documentation changes:

- [x] This pull request includes redirect settings in `mint.json` for files that are removed or renamed.

### CODE REVIEW

File: docs/3.0rc/react/incidents.mdx [hunk 1]
```diff
@@ -0,0 +1,95 @@
+---
+title: Incidents
+description: Incidents in Prefect Cloud help identify, resolve, and document issues in mission-critical workflows.
+---
+
+Incidents are a Prefect Cloud feature to help your team manage workflow disruptions. Incidents help you identify, resolve, and document issues with mission-critical workflows. This system enhances operational efficiency by automating the incident management process and providing a centralized platform for collaboration and compliance.
```
Comment (billpalombi) [inline docs/3.0rc/react/incidents.mdx:6]:
```suggestion
Incidents help you manage inevitable workflow disruptions, minimize their impact, and ensure operational continuity. With incidents, you can identify, resolve, and document issues with workflows, facilitating collaboration and compliance.
```

File: docs/3.0rc/react/incidents.mdx [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+
+## Definition
+
+Incidents are formal declarations of disruptions to a workspace. With [automations](#incident-automations), activity in a workspace can be paused when an incident is created and resumed when the incident is resolved.
```
Comment (billpalombi) [inline docs/3.0rc/react/incidents.mdx:10]:
```suggestion
Incidents vary in nature and severity, ranging from minor glitches to critical system failures. With [automations](#incident-automations), activity in a workspace can be paused when an incident is created and resumed when the incident is resolved.
```

File: docs/3.0rc/react/incidents.mdx [hunk 3]
```diff
@@ -0,0 +1,95 @@
+---
+title: Incidents
+description: Incidents in Prefect Cloud help identify, resolve, and document issues in mission-critical workflows.
+---
+
+Incidents are a Prefect Cloud feature to help your team manage workflow disruptions. Incidents help you identify, resolve, and document issues with mission-critical workflows. This system enhances operational efficiency by automating the incident management process and providing a centralized platform for collaboration and compliance.
+
+## Definition
```
Comment (billpalombi) [inline docs/3.0rc/react/incidents.mdx:8]:
```suggestion
```

File: docs/3.0rc/react/incidents.mdx [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+
+Incidents are formal declarations of disruptions to a workspace. With [automations](#incident-automations), activity in a workspace can be paused when an incident is created and resumed when the incident is resolved.
+
+Incidents vary in nature and severity, ranging from minor glitches to critical system failures. Prefect Cloud enables users to effectively and automatically track and manage these incidents, ensuring minimal impact on operational continuity.
```
Comment (billpalombi) [inline docs/3.0rc/react/incidents.mdx:12]:
```suggestion
```

File: docs/3.0rc/react/incidents.mdx [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+
+![Incidents in the Prefect Cloud UI](/3.0rc/img/ui/incidents-dashboard.png)
+
+## Use cases
```
Comment (billpalombi) [inline docs/3.0rc/react/incidents.mdx:16]:
```suggestion
```

File: docs/3.0rc/react/incidents.mdx [hunk 6]
```diff
@@ -0,0 +1,95 @@
...
+
+Incorporating incidents into your Prefect workflow facilitates many common needs:
```
Comment (billpalombi) [inline docs/3.0rc/react/incidents.mdx:18]:
```suggestion
Incorporating incidents into your Prefect workflow facilitates:
```

File: docs/3.0rc/react/incidents.mdx [hunk 7]
```diff
@@ -0,0 +1,95 @@
...
+
+* **From the Incidents page**, click the **+** button, fill in required fields, and attach any Prefect resources related to your incident.
+
+* **From a flow run, work pool, or block**, from the failed flow run, click the menu button and select "Declare an incident". This method automatically links the resource. 
+
+* **Through an [automation](/3.0rc/react/understand-events/automations-triggers/)**, set up incident creation as an automated response to selected triggers.
+
+### Automate an incident
+
+Automations can be used for triggering an incident and for selecting actions to take when an incident is triggered. For example, a work pool status change could trigger the declaration of an incident, or a critical level incident could trigger a notification action.
```
Comment (billpalombi) [inline docs/3.0rc/react/incidents.mdx:46]:
```suggestion
Automations can be used for triggering an incident and for taking action when an incident is triggered. For example, a work pool status change could trigger the declaration of an incident, or a critical level incident could trigger a notification action.
```

File: docs/3.0rc/react/incidents.mdx [hunk 8]
```diff
@@ -0,0 +1,95 @@
...
+
+To automatically take action when an incident is declared, set up a custom trigger that listens for declaration events.
```
Comment (billpalombi) [inline docs/3.0rc/react/incidents.mdx:48]:
```suggestion
To automatically take action when an incident is declared, set up a custom trigger that listens for declaration events:
```

File: docs/3.0rc/react/incidents.mdx [hunk 9]
```diff
@@ -0,0 +1,95 @@
...
+
+Incidents are a Prefect Cloud feature to help your team manage workflow disruptions. Incidents help you identify, resolve, and document issues with mission-critical workflows. This system enhances operational efficiency by automating the incident management process and providing a centralized platform for collaboration and compliance.
+
+## Definition
+
+Incidents are formal declarations of disruptions to a workspace. With [automations](#incident-automations), activity in a workspace can be paused when an incident is created and resumed when the incident is resolved.
+
+Incidents vary in nature and severity, ranging from minor glitches to critical system failures. Prefect Cloud enables users to effectively and automatically track and manage these incidents, ensuring minimal impact on operational continuity.
+
+![Incidents in the Prefect Cloud UI](/3.0rc/img/ui/incidents-dashboard.png)
```
Comment (billpalombi) [inline docs/3.0rc/react/incidents.mdx:14]:
```suggestion
```

**Review by billpalombi — COMMENTED** [review]
I think the intro, definition, and use case sections can be consolidated and pared down a bit. If we remove the image, we can get most of the text above the fold.

**Review by discdiver — APPROVED** [review]
LGTM. Merged @billpalombi's suggestions b/c I know you're OOO, @LarryUllman .

**Pull request merged**